src/css/librarybrowser.css

```diff
@@ -149,27 +133,20 @@
 }
 
 .headerLeft {
-    display: flex;
-    -webkit-align-items: center;
+    display: flex;justify-content: left;
```

Comment:
The justify-content attribute can be removed here, it is also defined at the end of this rule.